corehq/apps/domain/project_access/middleware.py

```diff
@@ -10,6 +10,9 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         if getattr(request, 'couch_user', None) and request.couch_user.is_superuser \
                 and hasattr(request, 'domain'):
             return self.record_entry(request.domain, request.couch_user.username)
```

Comment:
Based on the comment for 337f07172aa92d1f20928746f72898435c6ef4ee I think you do want to include superusers, but won't this return statement mean superusers never reach the lines below? `record_entry` always returns `None`, so calling that instead of returning it should be fine.

It'd also be an improvement to rename `record_entry` to something like `record_superuser_entry` now that there's more than one kind of entry.